scrubble-client/package.json

```diff
@@ -13,6 +13,9 @@
   "license": "ISC",
   "devDependencies": {
     "@types/pixi.js": "^5.0.0",
+    "@webcomponents/webcomponentsjs": "^2.5.0",
```

Comment:
so i think this is the polyfill? which we maybe dont need if we dont care too much about older browser support, right? IDK if its causing issues, or if its just extra and can be uninstalled.